components/social-follow/social-follow.snip.html

```diff
@@ -18,6 +18,9 @@
 
 
 {{#social-follow}}
+{{#header}}
```

Comment:
If we were including social-follow directly in our templates, we could. However social-follow is included by sidebar which is included by navbar. I could add it to the sidebar component, but I don't see how that is any different to adding it here.

Which file should I add the title to?